frontend/src/main.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
 
 import { ENV } from '@common/enums/app/env.enum';
 import { Routing } from '@navigation/routing/routing';
 import * as Sentry from '@sentry/react';
 import { BrowserTracing } from '@sentry/tracing';
+
+import { store } from './store/store';
```

Comment:
Could you please replace all the relative `/store` imports with the `@store` alias?